python/flashinfer/decode.py

```diff
@@ -1404,10 +1404,10 @@ def run(
                 maybe_lse,
                 get_cuda_stream(device),
             )
-            out = (o, maybe_lse) if return_lse else (o,)
+            out = [o, maybe_lse] if return_lse else [o]
         if v_scale is not None:
             out[0] *= v_scale
 
-        return out if return_lse else out[0]
+        return tuple(out) if return_lse else out[0]
```

Comment:
I suppose we should still keep
```suggestion
        return out if return_lse else out[0]
```
to be consistent with the function signature?

Reply:
Yes it is irrelevant to the bug. The signature says `union(tensor, tuple(tensor, tensor))` so i tried to ensure the consistency.